7/slim/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.10-slim
+MAINTAINER Michael Hirschler <michael.vhirsch@gmail.com>
+
+RUN groupadd --gid 1000 thumbor \
+    && useradd --uid 1000 --gid thumbor --shell /bin/sh thumbor
+
+RUN mkdir /usr/local/thumbor
+ENV HOME /usr/local/thumbor
+ENV PATH $PATH:/usr/local/thumbor/.local/bin
+ENV OPTIMIZERS "['thumbor.optimizers.jpegtran']"
+
+RUN apt-get update \
+    && apt-get install --yes --quiet --no-install-recommends \
+        gcc g++ python3-dev python3-all-dev \
+        # libraries: pycurl deps (libcurl4 is required at runtime)
+        libcurl4-openssl-dev libgnutls28-dev libssl-dev libcurl4 \
+        # libraries: cairosvg deps (libcairo2 is required at runtime)
+        libffi-dev libcairo2 \
+        # extensions
+        libjpeg-progs gifsicle \
+    && pip install --user --no-cache-dir cairosvg pycurl pyexiv2 thumbor==7.0.1 envtpl==0.6.0 \
```

Comment:
`pyexiv2` is the old version, you should use `py3exiv2`

Reply:
Good catch, thanks!

Reply:
@marcelometal I took a look at the difference between `py3exiv2` and `pyexiv2`. Both are using python3, so it's fine for me. There is no "old" `pyexiv2`, since it's currently maintained and in active development. So this should be a safe decision to keep `pyexiv2`. Also thumbor is currently discussing to switch to `pyexiv2`: https://github.com/thumbor/thumbor/issues/1403

Reply:
Second guess: I'll just install py3exiv2 until thumbor supports it :-D